bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/BahmniPatientDaoImplLuceneIT.java

```diff
@@ -213,4 +212,43 @@ public void shouldNotReturnDuplicatePatientsEvenIfTwoIdentifiersMatches() {
         assertTrue(patient.getExtraIdentifiers().contains("200006"));
     }
 
+    @Test
+    public void shouldSearchSimilarPatientByPatientName() {
+        List<PatientResponse> patients = patientDao.getSimilarPatientsUsingLuceneSearch("Peet", "", "c36006e5-9fbb-4f20-866b-0ece245615a1", 5);
+        PatientResponse patient1 = patients.get(0);
```

Comment:
these 2 lines should move below the assertEquals(2, patients.size()), so we get a clean error message in case the patients returned are empty or only of length 1